src/components/_global/BalAsset/BalAssetSet.vue

```diff
@@ -1,29 +1,36 @@
 <template>
-  <div
-    class="relative"
-    :style="{
-      width: `${width}px`,
-      height: `${size}px`
-    }"
+  <template
+    v-for="(addressesChunk, addressChunkIndex) in addressesChunks"
+    :key="`addressChunk-${addressChunkIndex}`"
```

Comment:
@garethfuller should the key be unique like so? or is `addressChunkIndex` is enough to be the key?

Reply:
it should be fine for the key to be unique per component

Reply:
Yeah the index is enough on it's own here. https://v3.vuejs.org/guide/list.html#maintaining-state